README.md

```diff
@@ -405,6 +405,137 @@ There are multiple methods you can use for creating the exact `BlockEvent` you w
 - `addTransactions(txns)` This method adds the hashes of a spread list of transaction events at the end of `block.transactions` field in the event.
 - `addTransactionsHashes(hashes)` This method adds a hashes spread list to the end of `block.transactions` field in the event.
 
+### TestAlertEvent
+
+The concept of a ` TestAlertEvent` class does not actually exist. It was not implemented because the `forta-agent`library provides built-in static methods that serve the purpose that`TestAlertEvent` would have fulfilled.
```

Comment:
```suggestion
The concept of a `TestAlertEvent` class does not actually exist. It was not implemented because the `forta-agent` library provides built-in static methods that serve the purpose that `TestAlertEvent` would have fulfilled.
```